src/components/flat-table/flat-table.interaction.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { StoryObj, composeStory } from "@storybook/react";
+import { userEvent, within, expect, waitFor } from "@storybook/test";
+import { allowInteractions } from "../../../.storybook/interaction-toggle/reduced-motion";
+import DefaultDecorator from "../../../.storybook/utils/default-decorator";
+import { FlatTable } from "./flat-table.component";
+import meta, { DefaultStory, Paginated } from "./flat-table.stories";
+
+type Story = StoryObj<typeof FlatTable>;
+
+export default {
+  title: "Flat Table/Interactions",
+  component: FlatTable,
+  parameters: {
+    themeProvider: { chromatic: { theme: "sage" } },
+  },
+};
+
+export const FlatTableFocused: Story = {
+  ...DefaultStory,
+  play: async ({ canvasElement }) => {
+    if (!allowInteractions()) return;
+
+    const canvas = within(canvasElement);
+    const focusableTableContainer = canvas.getByTestId("flat-table-container");
+
+    await userEvent.tab();
+
+    await waitFor(() => expect(focusableTableContainer).toHaveFocus());
```

Comment:
question (non-blocking): do we need to use `waitFor` here, will `await expect` not suffice?

Reply:
Yeah, I think you're correct. A usual `await expect...` should work. I'll get this amended.